packs/ac-feats/Billowing_Wings_p83NzvyOMMa5Dgdh.json

```
@@ -0,0 +1,94 @@
+{
+  "name": "Billowing Wings",
+  "type": "feat",
+  "effects": [],
+  "system": {
+    "description": {
+      "gm": "",
+      "value": "<p>You’ve trained your winged companions to gather air in their wings, whipping up a concentrated gust of wind at opponents, potentially staggering them.</p><p>Your companions with wings gain a gust ranged unarmed attack, which deals @Damage[1d4[bludgeoning]] damage with a range of 30 feet and has the air and propulsive traits. On a critical hit, the target is pushed back 5 feet. This is forced movement.</p>"
+    },
+    "rules": [
+      {
+        "key": "Strike",
+        "category": "unarmed",
+        "damage": {
+          "base": {
+            "damageType": "bludgeoning",
+            "dice": 1,
+            "die": "d4"
+          }
+        },
+        "group": "brawling",
+        "label": "Gust",
```

```suggestion
        "label": "PF2E.BattleForm.Attack.Gust",
```
Making life a bit easier for the translators. The other stuff will get localized later as we keep revising the module.